introduction.md

```diff
@@ -1 +1,5 @@
 Hello there! I'm Dan, and I'm on the Java path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?